crates/sui-types/src/object.rs

```diff
@@ -509,6 +500,20 @@ impl Object {
         }
     }
 
+    pub fn with_object_owner_for_testing(id: ObjectID, owner: ObjectID) -> Self {
```

Comment:
Could we make this a test-only function?

Reply:
I think that is outside the scope of this PR

Reply:
Yeah, I did not realize it's used somewhere else. It would be good to have it cleaned up at some point, though.